futaba/cogs/info/core.py

```diff
@@ -187,3 +189,136 @@ def __init__(self, bot):
             ctx.send(embed=embed),
             Reactions.SUCCESS.add(ctx.message),
         )
+
+    @staticmethod
+    async def get_messages(channels, ids):
+        return await asyncio.gather(*[Info.get_message(channels, id) for id in ids])
+
+    @staticmethod
+    async def get_message(channels, id):
+        async def from_channel(channel, id):
+            try:
+                return await channel.get_message(id)
+            except discord.NotFound:
+                return None
+
+        results = await asyncio.gather(*[from_channel(channel, id) for channel in channels])
+        return first(results)
+
+    @commands.command(name='message', aliases=['findmsg', 'msg'])
+    @commands.guild_only()
+    async def message(self, ctx, *ids: int):
+        '''
+        Finds and prints the contents of the messages with the given IDs.
+        '''
+
+        logger.info("Finding message IDs for dump: %s", ids)
+
+        if not check_mod_perm(ctx) and len(ids) > 3:
+            ids = islice(ids, 0, 3)
+            await ctx.send(content='Too many messages requested, stopping at 3...')
+
+        def make_embed(message, id):
+            if message is None:
+                embed = discord.Embed(colour=discord.Colour.dark_red())
+                embed.description = f'No message with id `{id}` found'
+                embed.timestamp = discord.utils.snowflake_time(id)
+            else:
+                embed = discord.Embed(colour=discord.Colour.teal())
+                embed.description = message.content or None
+                embed.timestamp = message.created_at
+                embed.set_thumbnail(url=message.author.avatar_url)
+                user_discrim = f'{message.author.name}#{message.author.discriminator}'
+                embed.add_field(name='Sent by', value=f'{message.author.mention} ({user_discrim})')
+
+                if ctx.guild is not None:
+                    embed.add_field(name='Channel', value=message.channel.mention)
+
+                embed.add_field(name='Permalink', value=message.jump_url)
```

Comment:
I initially did that, however it does not have the desired effect of "jumping" to the message. It instead opens a new browser tab with that link.

Maybe, but I want a mention in the embed so you can click on the user and do right click actions.
What do you mean by "the url of the embed"?

Reply:
The url of embed is the link it goes to if you click the title however since you mentioned that the masked link did not work this might also have the same effect